pynest/nest/tests/test_iaf_psc_alpha_ps.py

```diff
@@ -0,0 +1,190 @@
+                if np.any(nonzero):
+                    rds = rds[nonzero] / np.abs(reference[record][nonzero])
+                # ignore events around spike times for V if it diverges
+                if record == "V_m" and params["Delta_T"] > 0.:
+                    spiking = (dmm[record] > V_lim)
+                    rds = rds[~spiking]
+                rel_diff[model][record] = np.average(rds)
+        return rel_diff
+
+    def test_closeness_same_timescales(self):
+        '''
+        Compare models for tau_syn_ex == tau_syn_in (should be equal).
+        '''
+        Ie = 370.
+
+        ref = nest.Create("iaf_psc_alpha_canon", params={"I_e": Ie})
+        new = nest.Create("iaf_psc_alpha_ps", params={"I_e": Ie})
+
+        espikes = nest.Create("spike_generator", params={"spike_times": [
+                              10., 100., 500.], "spike_weights": [20.]*3})
+        ispikes = nest.Create("spike_generator", params={"spike_times": [
+                              15., 99., 200.], "spike_weights": [-20.]*3})
+
+        nest.Connect(espikes, ref)
+        nest.Connect(ispikes, ref)
+        nest.Connect(espikes, new)
+        nest.Connect(ispikes, new)
+
+        mm_ref = nest.Create("multimeter", params={"record_from": ["V_m"]})
+        mm_new = nest.Create("multimeter", params={
+                             "record_from": ["V_m", "I_syn_ex", "I_syn_in"]})
+
+        nest.Connect(mm_ref, ref)
+        nest.Connect(mm_new, new)
+
+        nest.Simulate(1000.)
+
+        data_ref = nest.GetStatus(mm_ref, "events")[0]
+        data_new = nest.GetStatus(mm_new, "events")[0]
+
+        self.assertTrue(np.allclose(data_new["V_m"], data_ref["V_m"]))
+
+    def test_unequal_different_timescales(self):
+        '''
+        Check that the behavior differs if ``tau_syn_in != tau_syn_ex``
+        '''
+        Ie = 370.
+
+        ref = nest.Create("iaf_psc_alpha_canon", params={"I_e": Ie})
+        new = nest.Create("iaf_psc_alpha_ps", params={
+                          "I_e": Ie, "tau_syn_in": 5.})
+
+        espikes = nest.Create("spike_generator", params={"spike_times": [
+                              10., 100., 500.], "spike_weights": [20.]*3})
+        ispikes = nest.Create("spike_generator", params={
+                              "spike_times": [500.], "spike_weights": [-20.]})
+
+        nest.Connect(espikes, ref)
+        nest.Connect(ispikes, ref)
+        nest.Connect(espikes, new)
+        nest.Connect(ispikes, new)
+
+        mm_ref = nest.Create("multimeter", params={"record_from": ["V_m"]})
+        mm_new = nest.Create("multimeter", params={
+                             "record_from": ["V_m", "I_syn_ex", "I_syn_in"]})
+
+        nest.Connect(mm_ref, ref)
+        nest.Connect(mm_new, new)
+
+        nest.Simulate(1000.)
+
+        data_ref = nest.GetStatus(mm_ref, "events")[0]
+        data_new = nest.GetStatus(mm_new, "events")[0]
+
+        # test before 500.
+        keep = data_new["times"] < 500.
```

Comment:
keep seems a strange name. Could you add a comment on why you expect equality before 500, but not thereafter?